Hazel/vendor/Glad/include/glad/glad.h

```diff
@@ -1,11 +1,11 @@
 /*
 
-    OpenGL loader generated by glad 0.1.28 on Sat Jan  5 02:36:41 2019.
+    OpenGL loader generated by glad 0.1.30 on Sun Jun  2 08:10:54 2019.
 
     Language/Generator: C/C++
     Specification: gl
-    APIs: gl=4.6
-    Profile: core
+    APIs: gl=4.5
+    Profile: compatibility
```

Comment:
It was good to update glad to gl 4.5, however we should stick to using the core profile.

Reply:
Resolved in https://github.com/TheCherno/Hazel/pull/82/commits/bdbd7f92c9bf8efa0d6ad248841a319e69ebc700